src/shared/timeseries.js

```diff
@@ -226,9 +226,11 @@ async function generateTimeseries(options = {}) {
   await fs.writeJSON(path.join('dist', 'features.json'), featureCollection);
 
   const { locations, timeseriesByDate } = transform.transposeTimeseries(timeseriesByLocation);
-  await fs.writeFile(path.join('dist', `timeseries.json`), JSON.stringify(timeseriesByDate, null, 2));
-  await fs.writeFile(path.join('dist', `locations.json`), JSON.stringify(locations, null, 2));
-
+  const combineLocationsTimeseriesByDate = { locations, timeseriesByDate };
```

Comment:
Maybe the prop should just be `timeseries` inside of here? Or maybe `dates`?

Thinking about how I'd iterate over it:

```js
const timeseries = getJSON('timeseries.json);
for (let date of timeseries.dates) {
  for (let locationId of date) {
    let location = timeseries.locations[locationId]
    // ... do stuff
  }
}
```

That doesn't feel great either. Thoughts?

Reply:
"Think that'll upset the masses?"

Yikes... I can't say for sure:-)

Reply:
> Maybe the prop should just be `timeseries` inside of here? Or maybe `dates`?
> 
> Thinking about how I'd iterate over it:
> 
> ```js
> const timeseries = getJSON('timeseries.json');
> for (let date of timeseries.dates) {
>   for (let locationId of date) {
>     let location = timeseries.locations[locationId]
>     // ... do stuff
>   }
> }
> ```
> 
> That doesn't feel great either. Thoughts?

..thinking....it seems like it would/could be a cleaner approach:-)

Reply:
I wonder if others have ideas for the names of the props? @chunder @qgolsteyn @camjc @ryanblock @NateBaldwinDesign @JacobDFrank?

Reply:
The above looks pretty good to me.
Would agree that the original name `combineLocationsTimeseriesByDate` isn't optimal anyhow; mainly because it feels like a function thanks to its first word being a verb.

Reply:
I see your point @camjc :-)